Assignments/stu6/assignments.py

```diff
@@ -12,27 +12,19 @@
 import time
 from datetime import datetime
 
-sqs = boto3.client('sqs')
-INSERT_CAT = ""
-SELECT_CAT = ""
-
-pg_pool = psycopg2.pool.SimpleConnectionPool(1, 20,
-                                             user="postgres",
-                                             password="Ihgdp51505150!",
-                                             host="localhost",
-                                             database="Cats")
+# sqs = boto3.client('sqs')
+# INSERT_CAT = ""
+# SELECT_CAT = ""
+#
+# pg_pool = psycopg2.pool.SimpleConnectionPool(1, 20,
+#                                              user="postgres",
+#                                              password="Ihgdp51505150!",
+#                                              host="localhost",
+#                                              database="Cats")
 
 
 def ex1():
-    print("TODO ...")
-    people_list = [
-        {'name': 'alice', 'age': 20, 'weight': 160, 'sex': 'male', 'id': 1},
-        {'name': 'bob', 'age': 10, 'weight': 130, 'sex': 'male', 'id': 2},
-        {'name': 'charlie', 'age': 15, 'weight': 120, 'sex': 'female', 'id': 3},
-    ]
-    # sort_people(people_list, 'weight', 'desc')
-    # print(people_list)
-
+    sort_people()
```

Comment:
This usage doesn't match the requirements:  `sort_people(people_list, 'weight', 'desc')`